docs/node/run-your-node/prerequisites/set-up-trusted-execution-environment-tee.md

```diff
@@ -223,9 +220,7 @@ For the legacy driver you need to specify `--device /dev/isgx` instead.
 
 :::tip
 
-Make sure to use the correct docker image based on your attestation method.
-For DCAP use the `ghcr.io/oasisprotocol/aesmd-dcap:master` and for EPID use the
-`ghcr.io/oasisprotocol/aesmd-epid:master` image.
+For DCAP use the `ghcr.io/oasisprotocol/aesmd-dcap:master` docker image.
```

Comment:
This tip is no longer needed as the example already shows the use of this image. Just remove it.